build/lib/uses-babel.coffee

```diff
@@ -0,0 +1,18 @@
+fs = require 'fs'
+
+BABEL_PREFIXES = [
+	"'use babel'"
```

Comment:
Oh damn, thanks. I had turned of `softTabs` for testing something, and was accidentally sprinkling hard tabs everywhere.